_posts/2023-06-16-childcare-lactation.md

```diff
@@ -0,0 +1,24 @@
+---
+author: Jason Judkins, Metta Rolando
+category: General
+date: 2023-06-16 12:00:00
+image: /static/img/blog/childcare-lactation-family.png
+layout: post
+post_photo_alt: DjangoCon US 2023 Childcare, Lactation, and Family
+post_photo_url: /static/img/blog/childcare-lactation-family.png
+title: WANTED! Parents at DjangoCon US 2023!
+---
+
+# Are you a working parent? Are you a nursing parent? We want you to attend DjangoCon US! 
+
+## Inclusion in the DjangoCon US space matters! 
```

Comment:
What is the reasoning behind having this as a header? As a reader, I felt like there was a missing paragraph explaining or elaborating. I'm not saying we should change it, only looking to understand.